### PR TITLE
Ember break out

### PR DESCRIPTION
**Pull request by ccb3**

### CODE REVIEW

File: dts/ember/0000-sparx5-135-emmc/device-tree.dts [hunk 1]
```diff
@@ -28,7 +28,7 @@
 &spi0 {
 	pinctrl-0 = <&si2_pins>;
 	pinctrl-names = "default";
-	status = "okay";
+	status = "okay"
```
Comment (troglobit) [inline dts/ember/0000-sparx5-135-emmc/device-tree.dts:31]: Why drop the semicolon?

File: board/ember/linux_defconfig [hunk 2]
```diff
@@ -0,0 +1,476 @@
+CONFIG_SYSVIPC=y
```
Comment (troglobit) [inline board/ember/linux_defconfig:1]: This file looks huge ... is it really a saved `make linux-update-defconfig` file or a saved `.config`?  To be able to track changes between different archs, and try to keep them feature wise in sync, we used to save in defconfig style instead, which is a LOT smaller.

I may be wrong here, so treat this as an fyi and minor reflection from my side.

Comment (ccb3) [reply board/ember/linux_defconfig:1]: It is pretty much a copy of the envoys linux_defconfig but perhaps that should be looked in to as well.

Comment (troglobit) [reply board/ember/linux_defconfig:1]: Yeah, please do, for your own sake.  Iirc that too is wrong.  Going forward you'll be much happier with the minimized version as the origin :)

File: dts/ember/0000-sparx5-135-emmc/device-tree.dts (hunk 1 above)
Comment (troglobit) [reply dts/ember/0000-sparx5-135-emmc/device-tree.dts:31]: That was the only thing that stuck out for me, I'll leave the rest of the dts review to @joazet

**Review by joazet — APPROVED** [review]
Reviewed-by: Joacim Zetterling <joacim.zetterling@westermo.com>

**Review by joazet — APPROVED** [review]
Reviewed-by: Joacim Zetterling <joacim.zetterling@westermo.com>

**Review by joazet — APPROVED** [review]
Reviewed-by: Joacim Zetterling <joacim.zetterling@westermo.com>

**Pull request merged**